app/src/main/java/org/simple/clinic/appupdate/AppUpdateModule.kt

```diff
@@ -32,4 +37,19 @@ open class AppUpdateModule {
         features = features
     )
   }
+
+  @Provides
+  fun providesAppUpdateHelpContactBasedOnCountry(
+      remoteConfigReader: ConfigReader,
+      moshi: Moshi,
+      country: Country
+  ): Optional<AppUpdateHelpContact> {
+    val type = Types.newParameterizedType(Map::class.java, String::class.java, AppUpdateHelpContact::class.java)
+    val configAdapter = moshi.adapter<Map<String, AppUpdateHelpContact>>(type)
+
+    val videoJson = remoteConfigReader.string("app_update_help_contact", "")
```

Comment:
Can we update the naming to something more descriptive? `videoJson` does not say much

Reply:
Ah, right. I forgot to change it when copying the code over.